deployment/terraform/assets/cluster.tf

```
@@ -82,6 +82,47 @@ resource "aws_instance" "app_server" {
   }
 }
 
+data "aws_vpc" "selected" {
+  tags = {
+    Name = "Default VPC"
+  }
+}
+
+data "aws_subnets" "loadtest_subnets" {
+  filter {
+    name = "vpc-id"
+    values = [data.aws_vpc.selected.id]
+  }
+
+  tags = {
+    Name = "loadtest*"
+  }
+}
+
+resource "aws_efs_file_system" "efs_shared" {
+  count = var.create_efs ? 1 : 0
+
+  tags = {
+   Name = "Shared-fs-${var.cluster_name}"
```

Following the convention for the `Name` tag, this should be something like
```suggestion
   Name = "${var.cluster_name}-shared-fs"
```